packages/framework/src/Support/Includes.php

```diff
@@ -42,70 +44,75 @@ public static function path(?string $filename = null): string
-            return $default === null ? null : $default;
+        if (! Filesystem::exists($path)) {
+            return $default === null ? null : new HtmlString($default);
         }
 
-        return file_get_contents($path);
+        return new HtmlString(Filesystem::get($path));
     }
 
     /**
      * Get the rendered Markdown of a partial file in the includes directory.
      *
      * @param  string  $filename  The name of the partial file, with or without the extension.
      * @param  string|null  $default  The default value to return if the partial is not found.
-     * @return string|null The rendered contents of the partial file, or the default value if not found.
+     * @return HtmlString|null The rendered contents of the partial file, or the default value if not found.
      */
-    public static function markdown(string $filename, ?string $default = null): ?string
+    public static function markdown(string $filename, ?string $default = null): ?HtmlString
     {
-        $path = static::path(basename($filename, '.md').'.md');
+        $path = static::normalizePath($filename, '.md');
 
-        if (! file_exists($path)) {
-            return $default === null ? null : Markdown::render($default);
+        if (! Filesystem::exists($path)) {
+            return $default === null ? null : new HtmlString(trim(Markdown::render($default, MarkdownDocument::class)));
         }
 
-        return Markdown::render(file_get_contents($path));
+        return new HtmlString(trim(Markdown::render(Filesystem::get($path), MarkdownDocument::class)));
     }
 
     /**
      * Get the rendered Blade of a partial file in the includes directory.
      *
      * @param  string  $filename  The name of the partial file, with or without the extension.
      * @param  string|null  $default  The default value to return if the partial is not found.
-     * @return string|null The rendered contents of the partial file, or the default value if not found.
+     * @return HtmlString|null The rendered contents of the partial file, or the default value if not found.
      */
-    public static function blade(string $filename, ?string $default = null): ?string
+    public static function blade(string $filename, ?string $default = null): ?HtmlString
     {
-        $path = static::path(basename($filename, '.blade.php').'.blade.php');
+        $path = static::normalizePath($filename, '.blade.php');
 
-        if (! file_exists($path)) {
-            return $default === null ? null : Blade::render($default);
+        if (! Filesystem::exists($path)) {
+            return $default === null ? null : new HtmlString(Blade::render($default));
         }
 
-        return Blade::render(file_get_contents($path));
+        return new HtmlString(Blade::render(Filesystem::get($path)));
+    }
+
+    protected static function normalizePath(string $filename, string $extension): string
+    {
+        return static::path(basename($filename, $extension).$extension);
```

Comment:
Thinking if we should support using subdirectories (could be done by removing `basename`), but I think that might be an anti-pattern, as if you need grouped data you probably want to use a data collection instead.